Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -63,6 +63,12 @@ installation of the ``kube-proxy`` add-on:
       kubectl -n kube-system delete ds kube-proxy
       iptables-restore <(iptables-save | grep -v KUBE)
 
+    Delete ``kube-proxy`` ConfigMap to make sure kube-proxy is not reinstalled during kubeadm upgrade (only works for >= 1.19):
```

Comment:
I've just rendered the docs (`make render-docs`), and the new instructions appeared in the wrong section. You need to move them to the `  .. group-tab:: K8s 1.16 and newer` tab above.

Reply:
I thought I quickly update the documentation using the Github edit button, and found out it isn't that easy :).
It was in the wrong tab but also wrong section (about new installations and not existing installations).

Moved the command 2 lines down to "existing installation" as additional command. 
Right now this works for upgrades on k8s 1.19 and newer but will change again in the future releases. 
No need to confuse users too much with the details I think. See also: https://github.com/kubernetes/kubernetes/pull/89593